src/GameLoop.tsx

```diff
@@ -534,14 +532,37 @@ export function GameLoop() {
     }
 
     return (
-        <BoardStyled>
-            <PlayerComponent
-                isVisible={isPlayerVisible}
-                adjustedCells={playerPhysics.adjustedCells}
-            />
-            <BoardComponent
-                boardCellMatrix={boardPhysics.boardCellMatrix}
-            />
-        </BoardStyled>
+        <>
+            <div style={AppStyle}>
+                <BoardStyled>
+                    <PlayerComponent
+                        isVisible={isPlayerVisible}
+                        adjustedCells={playerPhysics.adjustedCells}
+                    />
+                    <BoardComponent
+                        boardCellMatrix={boardPhysics.boardCellMatrix}
+                    />
+                </BoardStyled>
+                <WordList displayedWords={matchedWords} />
+            </div>
+        </>
     );
 }
+
+const wordStyle = {
+    display: "block",
+    background: "yellow",
+};
+
+const WordList = React.memo(({ displayedWords }) => {
```

Comment:
Why do you need `React.memo`?

Reply:
So normally a if the state of a React component changes, it re-renders all children components... unless the child uses `React.memo`, in which case the child only re-renders  if its `props` changes